app/routes/image-set.js

```diff
@@ -3,6 +3,17 @@ import OrganizationRouteMixin from 'lion-guardians/mixins/organization-route';
 import RequireUserMixin from 'lion-guardians/mixins/require-user';
 
 export default Ember.Route.extend( OrganizationRouteMixin, RequireUserMixin, {
+  beforeModel: function() {
+    var route = this;
+    return this._super().then(function() {
+      if (window.SKIP_MAPS) {
+        return Ember.RSVP.resolve();
```

Comment:
you do not need to return a resolved promise inside this function. You are already in "promise land" at this point and you only need to return if you are doing further async (like loading a map, or making an ajax call)